src/index.js

```diff
@@ -291,6 +294,20 @@ export default class SauceLabs {
         })
     }
 
+    async _getLatestSauceConnectVersion () {
+        const uri = SAUCE_CONNECT_VERSIONS_ENDPOINT
+        const responseType = 'json'
+
```

Comment:
No need to define them into const as we don't modify them here.

Reply:
True, just inlined them!